src/main/java/com/internet/shop/util/HashUtil.java

```diff
@@ -29,6 +32,7 @@ public static String hashPassword(String password, byte[] salt) {
                 hashedPassword.append(String.format("%02x", element));
             }
         } catch (NoSuchAlgorithmException exception) {
+            PropertyConfigurator.configure(LOG4J_PROPERTIES);
```

Comment:
let's remove this from here and move the path to logging file to .properties
https://mkyong.com/logging/log4j-hello-world-example/
here you can find an example of the .properties file with the path